### ISSUE TITLE
Spurious table data is rendered before table

### ISSUE THREAD

**Issue by twpayne** [charmbracelet/glamour#44]
The input Markdown:

```
## Pre-built Linux packages

Download a package for your operating system and architecture and install it
with your package manager.

| Distribution | Architectures                                             | Package                                                                   |
| ------------ | --------------------------------------------------------- | ------------------------------------------------------------------------- |
| Debian       | `amd64`, `arm64`, `armel`, `i386`, `ppc64`, `ppc64le`     | [`deb`](https://github.com/twpayne/chezmoi/releases/latest)               |
| RedHat       | `aarch64`, `armhfp`, `i686`, `ppc64`, `ppc64le`, `x86_64` | [`rpm`](https://github.com/twpayne/chezmoi/releases/latest)               |
| OpenSUSE     | `aarch64`, `armhfp`, `i686`, `ppc64`, `ppc64le`, `x86_64` | [`rpm`](https://github.com/twpayne/chezmoi/releases/latest)               |
| Ubuntu       | `amd64`, `arm64`, `armel`, `i386`, `ppc64`, `ppc64le`     | [`deb`](https://github.com/twpayne/chezmoi/releases/latest)               |
```

with the default `dark` theme and with `ansi.Options.WordWrap = 80` is rendered as:

<img width="1066" alt="glamour-wide-table" src="https://user-images.githubusercontent.com/6942/71526194-987da800-28cd-11ea-9881-890f16dc76f4.png">

Data from the table is rendered twice, and this seems to throw off the column alignment.

The original Markdown file that demonstrates this can be found [here](https://raw.githubusercontent.com/twpayne/chezmoi/a402f9eba1245b45de1aa0b05dda4b35c164f345/docs/INSTALL.md)

**Comment by twpayne** [comment]
This likely the same bug as https://github.com/charmbracelet/glow/issues/111.

**Comment by muesli** [comment]
The spurious output is fixed by 5a1a0f6.

To render AST children (like links) within table cells we still need to change the table renderer a bit. This is sadly not as trivial or straight-forward as I'd like, because the table rendering (by tablewriter) doesn't properly handle ANSI escape sequences. I'll probably move that to a separate ticket.
